### PR TITLE
Misc fixes and improvements

### PR DESCRIPTION
**Pull request by Nuckyz**

### CODE REVIEW

**Comment by Nuckyz** [comment]
I did a dumb I think, will fix later

**Comment by Nuckyz** [comment]
okay ready

File: src/plugins/showHiddenChannels/index.tsx [hunk 1]
```diff
@@ -239,17 +239,17 @@ export default definePlugin({
         {
             find: 'jumboable?"jumbo":"default"',
             replacement: {
-                match: /(?<=\i:\(\)=>\i)(?=}.+?(?<component>\i)=function.{1,20}node,\i=\i.isInteracting)/,
-                replace: ",hc1:()=>$<component>" // Blame Ven length check for the small name :pensive_cry:
+                match: /(?<=(?<component>\i)=function.{1,20}node,\i=\i.isInteracting.+?}}\)},)/,
+                replace: "shcEmojiComponentExport=($self.setEmojiComponent($<component>),void 0),"
```
Comment (Vendicated) [inline src/plugins/showHiddenChannels/index.tsx:243]: this seems unused?

Comment (Vendicated) [reply src/plugins/showHiddenChannels/index.tsx:243]: (the variable shcEmojiComponentExport)

Comment (Nuckyz) [reply src/plugins/showHiddenChannels/index.tsx:243]: Yes, it's just to be able to export the thing there and not a thousand lines under

**Pull request merged**